frontend/app/views/fragments/form/submit.scala.html

```diff
@@ -37,7 +39,14 @@
             <noscript>
                 <a href="http://www.enable-javascript.com/" target="_blank" class="action is-disabled">Enable javascript to proceed</a>.
             </noscript>
-            <button type="button" class="action js-submit-input u-hidden-non-js">
+            <button type="button" class="action js-submit-input u-hidden-non-js"
+                    @if(dataCategory != None && dataLabel != None) {
```

Comment:
Apologies, for Twirl templates the syntax looks more like this:

```
@for(label <- dataLabel; category <- dataCategory) {
  data-metric-category=@category
  data-metric-label=@dataLabel
}
```

...there's an existing example here: https://github.com/guardian/membership-frontend/blob/cb9cf5bfce/frontend/app/views/fragments/content/articleSnapshot.scala.html#L11